.devcontainer/devcontainer.json

```diff
@@ -5,27 +5,34 @@
   "build": {
     "dockerfile": "Dockerfile"
   },
-
+  "mounts": [
+    "source=${localWorkspaceFolderBasename}-node_modules,target=${containerWorkspaceFolder}/node_modules,type=volume"
+  ],
   "settings": {},
-
   // Add the IDs of extensions you want installed when the container is created.
   "extensions": [
     "dbaeumer.vscode-eslint",
     "esbenp.prettier-vscode",
     "ms-azuretools.vscode-docker",
     "mhutchie.git-graph"
   ],
-
   "forwardPorts": [
     // Application and Storybook
-    3000, 6006,
+    3000,
+    6006,
     // Local Firebase Emulator Suite
-    4000, 4400, 4500, 5000, 5001, 8080, 9099, 9150, 9199
+    4000,
+    4400,
+    4500,
+    5000,
+    5001,
+    8080,
+    9099,
+    9150,
+    9199
   ],
-
   // Use 'postCreateCommand' to run commands after the container is created.
   "postCreateCommand": "./.devcontainer/scripts/start.sh",
-
   // Comment out to connect as root instead. More info: https://aka.ms/vscode-remote/containers/non-root.
-  "remoteUser": "node"
-}
+  "remoteUser": "root"
```

Comment:
Let's just comment this out (per the comment directly above) rather than changing the user to `root`, that way if somebody wants to revert to non-root user locally, they have that option (i.e., might not be immediately to change it over to `node` otherwise)

Reply:
I take it back, that didn't work for me, it kept rebuilding the container as user `node`. I modified the comment (line `40`) accordingly to split the difference...